recipe/meta.yaml

```diff
@@ -1,21 +1,24 @@
 {% set name = "libgsasl" %}
-{% set version = "1.8.1" %}
+{% set version = "1.10.0" %}
 
 package:
   name: "{{ name }}"
   version: "{{ version }}"
 
 source:
   url: https://ftp.gnu.org/gnu/gsasl/{{ name }}-{{ version }}.tar.gz
-  sha256: 19e2f90525c531010918c50bb1febef0d7115d620150cc66153b9ce73ff814e6
+  sha256: f1b553384dedbd87478449775546a358d6f5140c15cccc8fb574136fdc77329f
 
 build:
   number: 0
   # This package is a dependency of libhdfs3, which is primarily used with HDFS on Linux.
-  skip: True  # [win]
+  # It's currently not available on s390x and there is no libgcrypt on s390x.
+  skip: True  # [win or s390x]
   run_exports:
     # unknown.  Leaving defaults.
     - {{ pin_subpackage('libgsasl') }}
+  ignore_run_exports:
+    - libgcrypt
```

Comment:
This `ignore run exports` seems suspect to me. It may be a sign that either the configure or link steps are not doing the right thing.

Reply:
I will dig deeper into this issue

Reply:
I found only this commit about `libgcrypt` https://git.savannah.gnu.org/gitweb/?p=gsasl.git;a=blob;f=lib/m4/gc.m4;h=0180d32b78fd48a8dd7ec73ba749718eefcf6809;hb=0180d32b78fd48a8dd7ec73ba749718eefcf6809

Reply:
After updating configure with `--with-libgcrypt`:
 `ERROR (libgsasl,lib/libgsasl.so.7.10.0): .. but ['libgpg-error'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)`

Reply:
@chenghlee Added in `run`:    
`- libgpg-error  # [linux]`

Now the build succeeded. But I have a question if  `libgcrypt-config` correctly defined:
```
checking for libgcrypt-config... $PREFIX/bin/libgcrypt-config
checking for LIBGCRYPT - version >= 1.4.4... yes (1.9.3)
configure: WARNING:
***
*** The config script "/tmp/build/80754af9/libgsasl_1642076888841/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/libgcrypt-config" was
*** built for x86_64-pc-linux-gnu and thus may not match the
*** used host x86_64-conda-linux-gnu.
*** You may want to use the configure option --with-libgcrypt-prefix
*** to specify a matching config script or use $SYSROOT.
***
checking for libgcrypt... yes
checking how to link with libgcrypt... $PREFIX/lib/libgcrypt.so $PREFIX/lib/libgpg-error.so -Wl,-rpath -Wl,$PREFIX/lib
```

Reply:
`$PREFIX/lib/libgcrypt.so $PREFIX/lib/libgpg-error.so` implies we should add `libgpg-error` to the `host` dependencies. We can then drop `libgpg-error` from the `run` dependencies, since that should be handled via run exports.

Reply:
Done